src/utils/vscodeUtils.ts

```diff
@@ -101,3 +104,25 @@ export function getNodeEditorLabel(node: IAzureNode): string {
 function isAccountTreeItem(treeItem: IAzureTreeItem): boolean {
     return (treeItem instanceof MongoAccountTreeItem) || (treeItem instanceof DocDBAccountTreeItemBase);
 }
+
+export function getDocumentTreeItemLabel(document: IMongoDocument | RetrievedDocument): string {
+    for (let field of documentLabelFields) {
+        if (document.hasOwnProperty(field)) {
+            let value = document[field];
+            if (value) { //ignore if false-y value: null, undefined, "".
+                if (typeof value === "string") {
+                    return value;
+                } else if (typeof value === "number") {
+                    return value.toString();
+                } else { // A simple instanceOf check doesn't seem to work. I run into this issue detailed here :
+                    //https://libertyseeds.ca/2015/01/03/Mongo-ObjectID-fails-instanceof-type-test-in-nodeunit/
+                    const result = value.toString();
+                    if (!result.startsWith("[object ")) {
+                        return result;
+                    }
+                }
+            }
+        }
+    }
+    return document["_id"].toString();
```

Comment:
Can _id be undefined or null?

Reply:
I'd leave it as _id, since we'll want to change documentLabelFields to be user-settable anyway.  Looks like previous code had return document._id.toString(). Still, I'd rather be conservative and not assume this won't give a null ref exception.

I think that String(document._id) is safe for null/undefined?  Can you verify? Thx.